cfssl/assets/configuration/spec.yaml

```
@@ -0,0 +1,18 @@
+name: cfssl
+files:
+- name: cfssl.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/default
+  - template: instances
+    options:
+    - name: url
+      description: The URL to the cfssl endpoint
+      required: true
+      enabled: true
+      value:
+        type: string
+        example: https://localhost:8888/
+        display_default: null
+    - template: instances/default
```

A nit, since the check doesn't submit metrics, you can override the `metric_patterns` config option from the `instances/default` template to hide it from the conf.yaml.example file

```suggestion
    - template: instances/default
      overrides:
        metric_patterns.hidden: true
```

---

Done. Not sure whether I need to regenerate some files? I ran some of the `ddev` commands but got no difference

---

Got my answer, had to run `ddev validate config cfssl -s`

(Can’t these things be done by a pre-commit hook?)